sdks/java/core/src/test/java/com/google/cloud/dataflow/sdk/testing/PAssertTest.java

```diff
@@ -292,38 +289,23 @@ public void testContainsInAnyOrderFalse() throws Exception {
     Throwable exc = runExpectingAssertionFailure(pipeline);
```

Comment:
The comment two lines up (`// The service runner does not give an exception we can usefully inspect.`) seems out of date